src/Gateways/AbstractGateway.php

```diff
@@ -956,6 +959,11 @@ public function process_payment( $order_id ) {
 				$this->payment_action,
 				$order->get_transaction_id()
 			);
+			// If the order contains a subscription, add the muti-use token to the order meta for repeat payments.
+			if(function_exists("wcs_order_contains_subscription") && wcs_order_contains_subscription($order) && $is_successful){
```

Comment:
This does not respect the white spacing coding standards (https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#whitespace).
It needs to be rewritten to: if ( function_exists( 'wcs_order_contains_subscription' ) && wcs_order_contains_subscription( $order ) && $is_successful ) {